analysis/tests/src/CompletionJsx.res

```diff
@@ -61,3 +61,15 @@ module IntrinsicElementLowercase = {
 
 // <IntrinsicElementLowercase
 //                            ^com
+
+module MultiPropComp = {
+  type time = Now | Later
+  @react.component
+  let make = (~name, ~age, ~time: time) => {
+    ignore(time)
+    name ++ age
+  }
+}
+
+// <MultiPropComp name="Hello" time= age="35"
```

Comment:
can you add examples with prop punning, and where there is more than one prop assignment already?

Reply:
Done, check the latest commit.